scripts/example.ts

```diff
@@ -171,19 +210,40 @@ async function main() {
       dryRun,
     });
 
+    // TODO -- this seems like it might be framework specific. Should we search for a folder?
```

Comment:
You can do the following:
- Does `src/stories` exist? Then use it. Otherwise:
- Does `./stories` exist? Then use it. Otherwise:
- Throw error